src/pybind/mgr/dashboard_v2/tests/test_nodb.py

```diff
@@ -244,8 +244,9 @@ def get_all_objects(api_controller, query):
     def test_make_model_args(self):
         args = NodbModelTest.SimpleModel.make_model_args(dict(a=1, bad=3))
         self.assertEqual(args, dict(a=1))
-        args = NodbModelTest.SimpleModel.make_model_args(dict(a=1, bad=3), fields_force_none=['b'])
-        self.assertEqual(args, dict(a=1, b=None))
+        args = NodbModelTest.SimpleModel.make_model_args(dict(a=1, bad=3))
+        self.assertEqual(args, dict(a=1))
+        self.assertIsNone(NodbModelTest.SimpleModel(**args).b)
```

Comment:
Feels like you're simply testing this for the sake of ensuring that the unmodified variable `b` still has its initially assigned value `None`, and is not being exercised anywhere else. What's the benefit of doing this instead of simply checking whether `b` is empty? (I'm guessing "empty" for an `IntegerField` would likely be 0 or something.)

Reply:
This code tests this the loop in [models/nodb.py#L830-L834](https://github.com/openattic/ceph/blob/wip-mgr-dashboard_v2/src/pybind/mgr/dashboard_v2/models/nodb.py#L830-L834) Without this loop, accessing `b` will fail with an `AttributeError`. This line asserts this loop was executed.